docker-compose.local.yml

```diff
@@ -0,0 +1,35 @@
+version: '3.3'
+services:
+    web:
+        build: .
+        image: geokrety:local
+        ports:
+            - "8000:80"
+        volumes:
+            - ${PWD}/website/:/var/www/html/:rw
+            - ${PWD}/configs/konfig-mysql.docker.php:/var/www/html/templates/konfig-mysql.php:ro
+            - ${PWD}/configs/konfig-local.tmpl.php:/var/www/html/templates/konfig-local.php:ro
```

Comment:
I don't think we still need to bind mount config file. They now can be configured via environment variable since 617e8dd

https://github.com/geokrety/geokrety-website/blob/d526607261a76644c5e52853535e7cfb69fde1cb/website/templates/konfig.php#L8-L12

Reply:
I didn't know that :) Fixed